pkg/admissioncontroller/webhooks/admission/internaldomainsecret/admission.go

```diff
@@ -167,3 +181,10 @@ func (h *handler) internalDomainSecretExists(ctx context.Context) (bool, error)
 
 	return len(secrets.Items) > 0, nil
 }
+
+// withSeedNameSelector extends the given selector with spec.seedName=<seedName>
+func withSeedNameSelector(listOptions []client.ListOption, seedName string) []client.ListOption {
```

Comment:
I don't think this function is needed/it can be done inline to prevent unnecessary hopping.